### PR TITLE
Tomee 2498 - Translate to Spanish: mp-faulttolerance-fallback

### PR DESCRIPTION
**Pull request by gdiazs**

### CODE REVIEW

File: examples/mp-faulttolerance-fallback/README_es.adoc [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Las politicas del Fallback permiten configurar :
+
+* **value**: Una clase que implementa `FallbackHandler`
+* **fallbackMethod**: El método que será ejecutado.
+
+Los parámetros `value` and `fallbackMethod` no pueden ser específicados a la vez.
+
+Para más detalles revisar http://download.eclipse.org/microprofile/microprofile-fault-tolerance-1.1/microprofile-fault-tolerance-spec.html[Especificación] 
+
+== Ejemplpos
```
Comment (Daniel-Dos) [inline examples/mp-faulttolerance-fallback/README_es.adoc]: HI @gdiazs,
please fix this name .

thanks.

File: examples/mp-faulttolerance-fallback/README_es.adoc [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+:jbake-type: page
+:jbake-status: published
+
+Este es un ejemplo de como usar Microprofile @Fallback en TomEE.
+
+== Fallback Feature
+Fault Tolerance Fallback provee una alternativa en caso de fallo de una ejecución. Esta alternativa será llamada cuando 
+un `Retry` o `CircuitBreaker` ha fallado.
+
+Para usar esta funcionalidad es necesario anotar el metodo con  `@Fallback`.
```
Comment (cesarhernandezgt) [inline examples/mp-faulttolerance-fallback/README_es.adoc]: método

File: examples/mp-faulttolerance-fallback/README_es.adoc [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+Las politicas del Fallback permiten configurar :
+
+* **value**: Una clase que implementa `FallbackHandler`
+* **fallbackMethod**: El método que será ejecutado.
+
+Los parámetros `value` and `fallbackMethod` no pueden ser específicados a la vez.
```
Comment (cesarhernandezgt) [inline examples/mp-faulttolerance-fallback/README_es.adoc]: especificados

File: examples/mp-faulttolerance-fallback/README_es.adoc (hunk 1 above)
Comment (gdiazs) [reply examples/mp-faulttolerance-fallback/README_es.adoc]: Thanks

File: examples/mp-faulttolerance-fallback/README_es.adoc (hunk 2 above)
Comment (gdiazs) [reply examples/mp-faulttolerance-fallback/README_es.adoc]: thanks

File: examples/mp-faulttolerance-fallback/README_es.adoc (hunk 3 above)
Comment (gdiazs) [reply examples/mp-faulttolerance-fallback/README_es.adoc]: thanks

**Pull request merged**